R/pcens.R

```diff
@@ -13,29 +13,18 @@
 #' @export
 new_pcens <- function(
     pdist, dprimary, dprimary_args,
-    pdist_name = NULL,
-    dprimary_name = NULL, ...) {
-  if (is.null(pdist_name)) {
-    pdist_name <- .extract_function_name(substitute(pdist))
-  }
-  if (is.null(dprimary_name)) {
-    dprimary_name <- .extract_function_name(substitute(dprimary))
-  }
+    pdist_name = deprecated(), dprimary_name = deprecated(),
+    ...) {
 
+  .name_deprecation(pdist_name, dprimary_name)
```

Comment:
as in other comments I think this is effectively a hard depreciation but we need to keep the functionality working for at least one release